spec/lib/sarif/base_sarif_spec.rb

```diff
@@ -41,4 +41,65 @@
         "invocations" => [base_sarif.build_invocations(scan_report, false)] })
     end
   end
+
+  describe '#build_runs_object' do
+    context 'results object' do
+      it 'has suppressions objects for suppressed results' do
+        parsed_issue = {
+          id: 'SAL0002',
+          name: "Golang Error",
+          level: "NOTE",
+          details: 'error',
+          start_line: 1,
+          start_column: 1,
+          uri: '',
+          help_url: "https://github.com/coinbase/salus/blob/master/docs/salus_reports.md",
+          code: "",
+          suppressed: true
+        }
+        adapter = Sarif::GosecSarif.new(scan_report)
+        adapter.instance_variable_set(:@logs, [parsed_issue])
+        runs_object = adapter.build_runs_object(true)
+        expect(runs_object['results'][0]['suppressions'].nil?).to eq(false)
+      end
+
+      it 'does not contain suppressed object when suppressed key is not present' do
+        parsed_issue = {
+          id: 'SAL0002',
+          name: "Golang Error",
+          level: "NOTE",
+          details: 'error',
+          start_line: 1,
+          start_column: 1,
+          uri: '',
+          help_url: "https://github.com/coinbase/salus/blob/master/docs/salus_reports.md",
+          code: ""
+        }
+        adapter = Sarif::GosecSarif.new(scan_report)
+        adapter.instance_variable_set(:@logs, [parsed_issue])
+        runs_object = adapter.build_runs_object(true)
+        expect(runs_object['results'][0]['suppressions'].nil?).to eq(true)
+      end
+
+      it 'does not contain suppressed object when includes_suppresd config is false' do
+        parsed_issue = {
+          id: 'SAL0002',
+          name: "Golang Error",
+          level: "NOTE",
+          details: 'error',
+          start_line: 1,
+          start_column: 1,
+          uri: '',
+          help_url: "https://github.com/coinbase/salus/blob/master/docs/salus_reports.md",
+          code: "",
+          suppressed: true
+        }
+        adapter = Sarif::GosecSarif.new(scan_report)
+        adapter.instance_variable_set(:@logs, [parsed_issue])
+        adapter.instance_variable_set(:@config, { "include_suppressed": false }.stringify_keys)
```

Comment:
These three checks work... Can you copy the 3rd check and add a 4th for the case of
```adapter.instance_variable_set(:@config, { "include_suppressed": false }.stringify_keys)```